src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -25,6 +30,7 @@
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
         // TODO: add log message about successful fetched data from DB
+        logger.info("The data was fetched from DB successful. Params: userId = {}", userId);
```

Comment:
The logging message 'The data was fetched from DB successful. Params: userId = {}' is informative, but similar to the previous comment, consider the security implications of logging user IDs. Ensure that this information is not sensitive or consider logging a more generic message.